### PR TITLE
Dev ensprod 8249

### PR DESCRIPTION
**Pull request by dpopleton**
Everything is good. We can add more classes as need be.

### CODE REVIEW

File: src/ensembl/production/metadata/api.py [hunk 1]
```diff
@@ -154,16 +132,16 @@ def __init__(self, metadata_uri=None, taxonomy_uri=None):
         super().__init__(metadata_uri)
 
         if taxonomy_uri is None:
-            taxonomy_uri = config.TAXONOMY_URI
+            taxonomy_uri = get_taxonomy_uri()
         self.taxonomy_db = load_database(taxonomy_uri)
```
Comment (marcoooo) [inline src/ensembl/production/metadata/api.py:136]: I believe you removed the `load_database` method, should it be `DBConnection` then?

Comment (dpopleton) [reply src/ensembl/production/metadata/api.py:136]: Yes, it should. I rolled back a good chunk of my taxonomy changes and I just wanted to push these changes to have a baseline to reintroduce my changes as all of the tests work at this point.

**Comment by dpopleton** [comment]
Everything is in working order and we can add any new functions that we need for flask.
The tests are just using a database spun up by travis.

File: src/ensembl/production/metadata/api.py (hunk 1 above)
Comment (dpopleton) [reply src/ensembl/production/metadata/api.py]: Fully fixed

File: tests/test_api.py [hunk 2]
```diff
@@ -15,27 +15,86 @@
 from os.path import dirname
 from ensembl.production.metadata.api import *
 
-DB_NAME = 'sqlite:///' + dirname(__file__) + '/TEST.db'
+DB_NAME = 'mysql://root:@127.0.0.1:3306/ensembl_metadata_2020'
```
Comment (marcoooo) [inline tests/test_api.py:18]: Could you change test setup to use a "ensembl:ensembl" user instead?

**Review by marcoooo — CHANGES_REQUESTED** [review]

**Review by marcoooo — APPROVED** [review]

**Pull request merged**